src/mixins/component.js

```diff
@@ -1,6 +1,6 @@
 import suit from '../util/suit';
 
-export default {
+export const createAlgoliaMixin = ({ connector } = {}) => ({
```

Comment:
Once we have the dedicated `suit` mixin we can maybe throw an error when there is no connector. The mixin makes no sense without it.

Reply:
Actually I just thought about that, and it allows you to make a fully custom widget, using the helper like this (pushed an example)

Reply:
It's a bit risky though because you have to manage what should be removed etc.. when it's unmounted. But sill it's a great capability.